.github/workflows/pipeline.yml

```diff
@@ -15,6 +15,8 @@ env:
   DHUBP: ${{secrets.DHUBP}}
   IMAGE_NAME: ${{vars.IMAGE_NAME}}
   PROXY_ENDPOINT: ${{vars.PROXY_ENDPOINT}}
+  NEON_TESTS_ENDPOINT: ${{vars.NEON_TESTS_ENDPOINT}}
+  DOCKERHUB_ORG_NAME: "neonlabsorg" # ${{vars.DOCKERHUB_ORG_NAME}}
```

Comment:
I'm not sure, but our dockerhub organization == github organization, no?

Reply:
Yes, But we moved it to DOCKERHUB_ORG_NAME
fixed